pkg/controller/workload/workload_event.go

```diff
@@ -107,6 +108,30 @@ func (svc *ServiceEvent) processWorkloadResponse(rsp *service_discovery_v3.Delta
 	}
 }
 
+func deletePodFontend(uid uint32) error {
+	var (
+		err error
+		bk  = BackendKey{}
+		bv  = BackendValue{}
+		fk  = FrontendKey{}
+	)
+
+	bk.BackendUid = uid
+	if err = BackendLookup(&bk, &bv); err == nil {
```

Comment:
What to do if `err != nil`.
There is no need to delete it if it doesn't exist in the map, but what if it's because of some other error?
Or does it mean that no other error occurs except NotExist?

Reply:
if err != nil, means not exist, I think we can ignore this err.